package.json

```diff
@@ -30,6 +30,12 @@
     "@emotion/styled": "^11.10.5",
     "@ethersproject/address": "^5.6.1",
     "@ethersproject/providers": "^5.7.1",
+    "@graphql-mesh/cross-helpers": "^0.4.6",
+    "@graphql-mesh/http": "0.103.2",
+    "@graphql-mesh/runtime": "0.103.2",
+    "@graphql-mesh/types": "^0.102.2",
+    "@graphql-mesh/utils": "^0.102.2",
+    "@graphql-tools/utils": "^10.5.3",
```

Comment:
why do we need all these new dependencies @0xJem ?

Reply:
They're peer dependencies for the graphclient